src/lib/components/schedule/schedule-form-view.svelte

```diff
@@ -31,13 +33,18 @@
   let namespace = $page.params.namespace;
   let scheduleId = $page.params.schedule;
 
-  let title = schedule ? 'Edit Schedule' : 'Create Schedule';
-  let loadingText = schedule ? 'Editing Schedule...' : 'Creating Schedule...';
-  let backTitle = schedule ? 'Back to Schedule' : 'Back to Schedules';
+  let title = translate('schedules', schedule ? 'edit' : 'create');
+  let loadingText = translate('schedules', schedule ? 'editing' : 'creating');
+  let backTitle = translate(
+    'schedules',
+    schedule ? 'back-to-schedule' : 'back-to-schedules',
+  );
   let backHref = schedule
     ? routeForSchedule({ namespace, scheduleId })
     : routeForSchedules({ namespace });
-  let confirmText = schedule ? 'Save' : 'Create Schedule';
+  let confirmText = schedule
+    ? translate('save')
+    : translate('schedules', 'create');
```

Comment:
Same here

Reply:
Same response as above, the copy is "Create Schedule". If it were just "Create", def would have put it in common.